app/views/fae/pages/home.html.slim

```diff
@@ -1,25 +1,53 @@
 header.content-header.js-content-header.-dashboard
-  i.icon-home>
-  | &nbsp;&nbsp;Welcome,
-  strong< = current_user.full_name
+ i.icon-home>
+ | &nbsp;&nbsp;Welcome,
+ strong< = current_user.full_name
 
 == render 'flash_messages'
 
 main.content
-  table.js-sort-column
-    thead
-      tr
-        th Name
-        th Type
-        th.-action-wide Modified
-    tbody
-      - @list.each do |item|
-        - begin
-          - parent = item.respond_to?(:fae_parent) ? item.fae_parent : nil
-          - edit_path = edit_polymorphic_path([main_app, fae_scope, parent, item])
-          - index_path = polymorphic_path([main_app, fae_scope, parent, item.class.to_s.pluralize.underscore])
-          tr
-            td: a href=edit_path = item.fae_display_field
-            td: a href=index_path = item.class.to_s
-            td = fae_date_format item.updated_at
-        - rescue
+- if @list.empty?
+   section.content.tutorial
+     h2
+       | Let's get started!
+     p
+       | Using the Fae CMS, we can generate objects quickly leveraging the rails generator commands. Creating an object is easy!
+       pre
+         | rails g fae:scaffold ArticleCategory name:string position:integer
```

Comment:
I would prefer to keep real documentation out of this page. It's hard enough to keep one set of documentation up to date 😉. I think the messaging can be simpler and link off to our quickstart guide: https://www.faecms.com/documentation/quickstart-guide